src/Evolution/Systems/NewtonianEuler/ConservativeFromPrimitive.cpp

```diff
@@ -9,16 +9,18 @@
 #include "Utilities/GenerateInstantiations.hpp"
```

Comment:
For all these commits, please add a short description as to what is being update. Having a commit message that effectively says "I changed code" isn't very useful when one is trying to track down regressions or using `git blame`. Can you also say for which system? e.g.
```
Update cons from prim for Newtonian Euler

- Switch to camel case
- Remove `double` support (now only supports DataVector like GRMHD)
```